src/components/setup.svelte

```diff
@@ -21,6 +26,10 @@
 		await start();
 
 		gameStore.start();
```

Comment:
i think we should move this call also into the startFirstNightPhase function and then merge the `gameStore.setState('night')` into the `gameStore.start()` function